StoreAPI/Models/Order.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace StoreAPI.Models
+{
+    public class Order
+    {
+        public int OrderId { get; set; }
+        public DateTime OrderDate { get; set; }
+
+        public Product Product { get; set; }

```

Comment:
Order entity should not contain any info about products. Only OrderProduct knows about order.